tree/dataframe/inc/ROOT/RDF/RFilterBase.hxx

```diff
@@ -53,9 +53,14 @@ public:
    RFilterBase(RLoopManager *df, std::string_view name, const unsigned int nSlots,
                const RDFInternal::RColumnRegister &colRegister, const ColumnNames_t &columns,
                const std::vector<std::string> &prevVariations, const std::string &variation = "nominal");
-   RFilterBase &operator=(const RFilterBase &) = delete;
 
-   ~RFilterBase() override;
+   // Rule of five
+
+   RFilterBase(const RFilterBase &) = delete;
+   RFilterBase &operator=(const RFilterBase &) = delete;
+   RFilterBase(RFilterBase &&) = delete;
+   RFilterBase &operator=(RFilterBase &&) = delete;
+   ~RFilterBase() override = default;
```

Comment:
Often the out-line destructor is used to 'anchor' the virtual table generation into a specific object file.  Is that not in play here?

Reply:
It was not done for every class, I understand it might have had its use, so I can write the destructor out of line for all the classes touched by this PR